saleor/settings.py

```diff
@@ -340,7 +340,8 @@ def get_host():
 
 TEST_RUNNER = ''
 
-ALLOWED_HOSTS = get_list(os.environ.get('ALLOWED_HOSTS', 'localhost'))
+ALLOWED_HOSTS = get_list(os.environ.get('ALLOWED_HOSTS',
+                                        'localhost,127.0.0.1'))
```

Comment:
Could you put this `os.environ.get(...)` on a new line? Exactly like this:
```python
ALLOWED_HOSTS = get_list(
    os.environ.get('ALLOWED_HOSTS', 'localhost,127.0.0.1'))
```

Reply:
Sure